frontend/src/components/campoComBuscaRemota/CampoComBuscaRemota.vue

```diff
@@ -82,9 +84,10 @@
       v-else-if="buscaRealizada"
       class="tablemain"
     >
-      <thead v-if="linhas.length && $slots.tableHeader">
+      <thead v-if="linhas.length && $slots['table-header']">
         <tr>
-          <slot name="tableHeader" />
+          <slot name="table-header" />
```

Comment:
por que a troca do case? tem algum padrão que eu não to sabendo?

Reply:
o padrão de nomenclatura para slots com vue é kebab-case